project/Dependencies.scala

```diff
@@ -50,8 +49,14 @@ object Dependencies {
   val identityModel = ("com.gu.identity" %% "identity-model" % identityLibVersion)
     .excludeAll(excludeDirectScalaXMLDependency)
   val identityAuthPlay = ("com.gu.identity" %% "identity-auth-play" % identityLibVersion)
-    .excludeAll(excludeDirectScalaXMLDependency)
-
+    .excludeAll(
+      excludeDirectScalaXMLDependency,
+   // Temporarily excluding those so that we can get the Play 3 branch to compile.
+//      ExclusionRule("com.typesafe.play", "play_2.13"),
+//      ExclusionRule("com.typesafe.play", "twirl-api_2.13"),
+//      ExclusionRule("com.typesafe.play", "twirl-compiler_2.13"),
+//      ExclusionRule("com.typesafe.play", "twirl-parser_2.13"),
```

Comment:
@mkurz I commented these exclusions and I can't see any `com.typesafe.play` dependency being pulled in from anywhere? Does this mean this is good to go? Same for [lines 108-110](https://github.com/guardian/frontend/pull/26755/files#diff-ad2642dc77b3679e4ad2c7d3b2f59a2dcf48c1bf5b93a558a4fa44828315a34cR108-R110)

Reply:
Please wait with merging, there is still at least `com.typesafe.play:twirl-api_2.13:1.4.2` pulled in, which should not be the case.
I will investigate asap.

Reply:
Spoke too soon, sorry! I think the things we still need to fix are:

1. As you say: `com.typesafe.play:twirl-api_2.13:1.4.2`. This is coming from [`atom-renderer`](https://github.com/guardian/atom-renderer). It's a legacy repo we have. I can have a look, please feel free to contribute
<img width="479" alt="image" src="https://github.com/guardian/frontend/assets/19683595/9bfbbf96-6f67-4e0c-bbe7-57b9b74f586d">

2. identity project also pulls `com.typesafe.play` dependencies. We need to fix this. This is a Guardian private repo so you can't contribute but I can speak with the team and get back to you.

Also, I forgot to comment out the exclusion for `play-json-extensions` [here](https://github.com/guardian/frontend/pull/26755/files#diff-ad2642dc77b3679e4ad2c7d3b2f59a2dcf48c1bf5b93a558a4fa44828315a34cR103-R104) but just discovered we're not using this anywhere so raising a separate PR to remove it.

Reply:
Maybe I have time to look at this tomorrow?

Reply:
I merged #26983 and with the help of @kelvin-chappell from @guardian/identity team we no longer pull pre Play 3 dependencies from identity. The only place we still have is `atom-renderer`. I started a branch locally but it looks like I don't have permissions to push so need to sort this out. I'm on leave tomorrow but will have a look on Friday. In the meantime, please feel free to have a look. Locally I have only made 2 changes so far:

1. Changed this [line](https://github.com/guardian/atom-renderer/blob/master/project/plugins.sbt#L1) to `addSbtPlugin("org.playframework.twirl" % "sbt-twirl" % "2.0.4")`
2.  Bumped sbt to 1.9.9

but I'm still getting the old `twirl-api` somehow:

![image](https://github.com/guardian/frontend/assets/19683595/c97dc1f3-4c73-4988-9942-082e4cb80a38)

Reply:
@mkurz In case you were planning to, please hold off from doing any work on `atom-renderer`. I'm having a look at removing the dependency completely. This is a very old and unmaintained repo.

Reply:
@mkurz in case you were planning to work on updating `atom-renderer` please hold off. I am working on moving some of the code to frontend and decommissioning the repo.

Reply:
@ioannakok Actually I was trying to upgrade the `atom-renderer` repo locally (before easter) and almost could make it work with some hacks, however I have seen you already opened PRs to drop it, which probably is the better thing to do at this point :+1:

Reply:
@mkurz Yes, I've just merged the PR: 
* https://github.com/guardian/frontend/pull/27021

I have just rebased. I'm going to raise this PR today - need to add a few things in the description.

Reply:
ah sorry missed this comment, just commented way down below :wink: